### ISSUE TITLE
Using the WriteTo() method with go 1.21 got problems

### ISSUE THREAD

**Issue by ShowerBandV** [qax-os/excelize#1734]
**Description**
We used excelize in the project, and in our business scenario, I need to call the WriteTo() method to create an Excel file. When I use go 1.18.10.0, this is completely feasible. Recently, we switched to go 1.21.0.0 and found that the generated Excel file is damaged. After converting to an XML file, it was found that each tag contains the xmlns="" attribute. After careful investigation, we found that the cause should be the 1.21 version of go.

**Code And Result**
here is the code

```go
func main() {
	file := excelize.NewFile()
	buf := bytes.NewBuffer(nil)
	_, err := file.WriteTo(buf)
	if err != nil {
		log.Fatalf("write err:%s", err.Error())
	}
	openFile, err := os.OpenFile("empty.xlsx", os.O_CREATE|os.O_WRONLY, 0666)
	if err != nil {
		log.Fatalf("openfile err:%s", err.Error())
	}
	defer openFile.Close()
	n, err := buf.WriteTo(openFile)
	if err != nil {
		log.Fatalf("write file err:%s", err.Error())
	}
	fmt.Println("write bytes ", n)
}
```

This is the XML generated for Excel conversion in version 1.21

```xml
<worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships">
    <dimension xmlns="" ref="A1"/>
    <sheetViews xmlns="">
      <sheetView tabSelected="true" workbookViewId="0"/>
    </sheetViews>
    <sheetFormatPr xmlns="" defaultRowHeight="15"/>
    <sheetData xmlns=""/>
</worksheet>
```

This is the XML generated for Excel conversion in version 1.18

```xml
<worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships">
    <dimension ref="A1"/>
    <sheetViews>
      <sheetView tabSelected="true" workbookViewId="0"/>
    </sheetViews>
    <sheetFormatPr defaultRowHeight="15"/>
    <sheetData/>
</worksheet>
```

The file generated by Go 21 version cannot be opened in Excel, and Excel will report an error that needs to be fixed

**Conclusion**

the diffrent is the `xmlns=""` attr and the go 1.21 version seems to be the reason，can you fix this bug? After all, it's due to the upgrade of the Go version

**Comment by ShowerBandV** [comment]
Thank you for reply. Hope it will be fixed soon

**Comment by xuri** [comment]
This issue already has been fixed in https://go.dev/cl/466295. As my reply in above, it has been fixed on Go 1.21.1. So please using the Go 1.20.11 and previous Go released version or upgrade to Go 1.21.1 and later.

**Comment by ShowerBandV** [comment]
ok,thank you
